pandas-stubs/core/generic.pyi

```diff
@@ -359,7 +360,7 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
         self, func: Callable[..., T] | tuple[Callable[..., T], str], *args, **kwargs
     ) -> T: ...
     def __finalize__(self, other, method=..., **kwargs) -> NDFrame: ...
-    def __getattr__(self, name: _str): ...
+    def __getattr__(self, name: _str) -> Series: ...
```

Comment:
Move this to `core/frame.pyi` and you can avoid the `# type: ignore` in `core/series.pyi`